docs/exceptions/README.md

```diff
@@ -7,13 +7,24 @@ path_base_for_github_subdir:
 
 # Semantic Conventions for Exceptions
 
-**Status**: [Stable][DocumentStatus]
-
-This document defines semantic conventions for Exceptions.
+**Status**: [Stable][DocumentStatus], Unless otherwise specified.
 
 Semantic conventions for Exceptions are defined for the following signals:
 
 * [Exceptions on spans](exceptions-spans.md): Semantic Conventions for Exceptions associated with *spans*.
 * [Exceptions in logs](exceptions-logs.md): Semantic Conventions for Exceptions recorded in *logs*.
 
+## Reporting errors in instrumentation code
+
+**Status**: [Development][DocumentStatus]
+
+When instrumented operation fails with an exception, instrumentation SHOULD record
+this exception as a [span event](exceptions-spans.md) or a [log record](exceptions-logs.md).
+
+It's NOT RECOMMENDED to record exceptions that are handled by the instrumented library.
```

Comment:
Maybe it is a good idea to include an example of such cases, to avoid any ambiguity/different in interpretations.